style.css

```diff
@@ -0,0 +1,62 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+section{
+    display: flex;
+    justify-content:center;
+    align-items: center;
+    min-height: 100vh;
+    background: #042104;
+    animation: animateBg 10s linear infinite;
+}
+@keyframes animateBg
+{
+    0%
+    {
+        filter:hue-rotate(0deg);

```

Comment:
what does it do ?

Reply:
it will pick color which is at 0deg of the color wheel .